gcovr/merging.py

```diff
@@ -328,6 +327,7 @@ def merge_call(
     Do not use 'left' or 'right' objects afterwards!
     """
     left.covered |= right.covered
+    left.callno = right.callno
```

Comment:
I think here it should be checked that left and right are equal (`assert`), else they can't be merged.

Reply:
Changed on 1b0f9e7